qiime2/plugins.py

```diff
@@ -37,6 +37,13 @@ def _init_metadata_(self, ref, factory):
         self._init_data_refs[ref] = factory
         return ref
 
+    def _init_data_collection_(self, ref, collection_type, *records):
+        self._init_data_refs[ref] = ref
```

Comment:
I don't think we need this line, since the individual records in the collection have already been registered. The `_init_data_refs` attribute is for tracking data that we might need to materialize, in case the user asks for it (via the `get_example_data()` method).

Reply:
right-o! 🌭